### ISSUE TITLE
capture currently visible and clicked image

### ISSUE THREAD

**Issue by yosiasz** [MurhafSousli/ngx-gallery#603]
Greetings!

What is the expected behavior?
capture the click event of the currently visible image 

What is the current behavior?
No sure

What are the steps to reproduce?
implemented `(click)=onClick($event)` but not sure how to manipulate dom to magnify image via dynamic class

What is the use-case or motivation for changing an existing behavior?
to be able to manipulate and customize style on gallery-image for example enlarge image for better viewing

Which versions are you using for the following packages?
Angular:16
Angular CDK:16.2.13
Angular CLI: 16.2.11
Typescript: 5.1.6
Gallery: 11.0.0

Is there anything else we should know?
tried using recommended zoom library but not fit for our use case

Thanks

**Comment by MurhafSousli** [comment]
You probably need to use a custom template, then you can use another library to maginfy your template like [√](https://github.com/MurhafSousli/ngx-gallery/wiki/Image-zoom-example)

**Comment by yosiasz** [comment]
@MurhafSousli thanks.
does the next beta version feature help with this?

feature: Introduce galleryImage directive within galleryItemDef, to allow recognizing the img element in your custom item template.

**Comment by MurhafSousli** [comment]
Yes this is already implemented in beta version, just add `galleryImage` directive to your `img` elements within your `galleryItemDef`
